ViewModelKit/VMKTableViewRowActionCreator.m

```diff
@@ -21,12 +21,61 @@ - (instancetype)initWithTableView:(UITableView *)tableView rowActionsType:(id<VM
     return self;
 }
 
+#pragma mark - UISwipeActionsConfiguration
+
+- (nullable UISwipeActionsConfiguration *)swipeActionsConfiguration {
+    
+    NSMutableArray<UIContextualAction *> * actions = [NSMutableArray array];
+    
+    for (VMKTableViewRowActionViewModel *rowAction in self.rowActionsType.rowActions) {
+        
+        UIContextualActionStyle style = [self contextualActionStyleFromRowActionViewModel: rowAction];
+        
+        UIContextualAction *tableViewAction = [UIContextualAction contextualActionWithStyle: style
+                                                                                      title: rowAction.title handler:^(UIContextualAction * _Nonnull action, __kindof UIView * _Nonnull sourceView, void (^ _Nonnull completionHandler)(BOOL)) {
+            VMKViewModel *viewModel = [rowAction swipedRowAction];
+            if (viewModel) {
+                // positions were voided
+                VMKTableViewDataSource *tableViewDataSource = self.tableView.dataSource;
+                BOOL shouldOverrideSourceViewForSwipeAction = [rowAction.delegate shouldOverrideSourceViewForSwipeAction];
+                UIView * cellView = sourceView.superview.superview;
+                UIView * viewToSend = shouldOverrideSourceViewForSwipeAction ? cellView : sourceView;
+                [tableViewDataSource requestViewWithViewModel:viewModel fromView: viewToSend];
+                // view model is returned => it's actually an alert view model
+                completionHandler(NO);
+            } else {
+                // positions were removed
+                completionHandler(YES);
+            }
+        }];
+        
+        tableViewAction.backgroundColor = rowAction.backgroundColor;
+        [actions addObject: tableViewAction];
+    }
+    
+    UISwipeActionsConfiguration * config = [UISwipeActionsConfiguration configurationWithActions: actions];
+    config.performsFirstActionWithFullSwipe = NO;
+    return config;
+}
+
+- (UIContextualActionStyle)contextualActionStyleFromRowActionViewModel:(VMKTableViewRowActionViewModel *)rowActionViewModel {
+    if (rowActionViewModel.style == VMKTableViewRowActionViewModelStyleDestructive) {
+        return UIContextualActionStyleDestructive;
+    }
+    return UIContextualActionStyleNormal;
+
+}
```

Comment:
Can we do this mapping in the same file where we have defined these style types so that we can avoid this mapping repeatedly? @tovnixon